CMakeLists.txt

```diff
@@ -191,12 +192,12 @@ endif()
 # We want Kokkos to be built with C++14, since that's what we're using in
 # Parthenon.
 set(Kokkos_CXX_STANDARD 14)
-if(EXISTS ${Kokkos_ROOT}/CMakeLists.txt)
+if(EXISTS ${EXTERNAL_KOKKOS})
```

Comment:
Is EXTERNAL_KOKKOS a path or a switch? I'm confused because you are using it I think as both.

Reply:
Yes I was using it as both, which seemed to work actually, but I've removed the `option(EXTERNAL_KOKKOS` because it should really just be a path.